packages/http/libraries/core/src/http/decorators/SetHeader.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  getReflectMetadata,
+  setReflectMetadata,
+} from '@inversifyjs/reflect-metadata-utils';
+
+import { controllerMethodHeaderMetadataReflectKey } from '../../reflectMetadata/data/controllerMethodHeaderMetadataReflectKey';
+import { ControllerFunction } from '../models/ControllerFunction';
+
+export function setHeader(key: string, value: string): MethodDecorator {
+  return (
+    _target: object,
+    _key: string | symbol,
+    descriptor: PropertyDescriptor,
+  ): void => {
+    const headerMetadata: Map<string, string> =
```

Comment:
Do we want to support multiple values in order to combine them? Refer to https://www.rfc-editor.org/rfc/rfc9110.html#name-field-lines-and-combined-fi for more information.

For example, developers might want to `@setHeader('Cache-Control', 'no-cache') @setHeader('Cache-Control', 'no-store')`